src/openrct2-ui/windows/Sign.cpp

```diff
@@ -232,6 +236,10 @@ static void window_sign_mouseup(rct_window* w, rct_widgetindex widgetIndex)
                     }
                 }
                 tile_element++;
+                if (tile_element >= &gTileElements[std::size(gTileElements)])
```

Comment:
This should only loop through tile elements at this coordinate. So we should break the loop and return if on last element.

Reply:
I have tried that, but that seems to cause _more_ crashes, since it then cannot find the large scenery element.

Reply:
But that doesn't make any sense. All of the tiles that comprise of the large scenery element will have the banner index set if it is a scrolling banner. How would the LargeSceneryRemoveAction work if the x,y coordinate is incorrect?